fud2/src/main.rs

```diff
@@ -419,23 +443,42 @@ fn build_driver() -> Driver {
         xclbin,
         vcd,
         |e, input, output| {
+            e.rsrc("xrt_trace.ini")?;
             e.build_cmd(
                 &[output], // TODO not the VCD, yet...
                 "xclrun",
                 &[input, "$sim_data"],
-                &["emconfig.json", "pre_sim.tcl", "post_sim.tcl"],
+                &[
+                    "emconfig.json",
+                    "pre_sim.tcl",
+                    "post_sim.tcl",
+                    "xrt_trace.ini",
+                ],
             )?;
-            let rsrc_dir = e.config_val("rsrc")?;
-            e.arg("xrt_ini", &format!("{}/xrt_trace.ini", rsrc_dir))?;
+            e.arg("xrt_ini", "xrt_trace.ini")?;
             Ok(())
         },
     );
-
-    bld.build()
 }
 
 fn main() -> anyhow::Result<()> {
-    let driver = build_driver();
+    let mut bld = DriverBuilder::new("fud2");
+    build_driver(&mut bld);
+
+    // In debug mode, get resources from the source directory.
+    #[cfg(debug_assertions)]
+    bld.rsrc_dir(manifest_dir_macros::directory_path!("rsrc"));
```

Comment:
Oh this is cool! We should probably do something similar to the Calyx compiler when pulling in primitives.

Reply:
That would be cool indeed!